contribs/vsp/src/main/java/org/matsim/contrib/freight/analysis/FreightAnalysisServiceTracking.java

```diff
@@ -87,19 +87,19 @@ public void setCalculatedArrival(Id<Carrier> carrierId, Id<CarrierService> servi
 	}
 
 	// UNTESTED handling of LSP Service events that provided reliable info about driver and timestamps.
```

Comment:
outdated comment since you erased all the lsp stuff

Reply:
Same here: Updating freight analysis will be a seperate step